for testing

kaiyork opened the pull request:
pulling main into kai branch so i can make use of the stuff for testing

CamFroth committed:
Committing to code on PC, not finished yet lol

CamFroth committed:
Added autolog and manual log pages to nav

Added modal to autolog before tracking begins

Next steps: adding settings and profile

CamFroth committed:
Added stylesheet to separate file to be imported
where necessary

See home.js for example

CamFroth committed:
Added notifications setting page
Added Download data setting page
Accessible through buttons on settings page

CamFroth committed:
Changed splash screen when compiling app

CamFroth committed:
Added toast message on press of download button

CamFroth committed:
Deleted download page
Added documentation

CamFroth committed:
Deleted Download page
Added documentation

camtheboom committed:
Merge pull request #36 from camtheboom/adding-pages-to-app

Page Navigation and Front End - Home Page and Settings Page

Jonaswang618 committed:
Create track position map view.js

Jonaswang618 committed:
Merge branch 'main' of https://github.com/camtheboom/CITS3200-Team-4

Jonaswang618 committed:
Create track and visualization.js

CamFroth committed:
Removed download.js and updated content on
travellog

CamFroth committed:
Edit

CamFroth committed:
Merge pull request #37 from camtheboom/adding-pages-to-app

Adding pages to app

Jonaswang25 committed:
Add files via upload

geolocation API

Jonaswang25 committed:
Delete track position map view

Jonaswang25 committed:
Delete track position map view.js

CamPBrown committed:
Merged frontend and backend together

CamPBrown committed:
Merge pull request #38 from camtheboom/master

Merged frontend and backend together

kaiyork reviewed (commented):
changing the files in kai testing branch with updated main

CamPBrown reviewed (approved):
All good, this will be very useful for testing how the backend and frontend behave with each other

kaiyork reviewed (commented):
merging

Jonaswang25 committed:
Add files via upload

Jonaswang25 committed:
Delete gps_tracker.java

CamPBrown committed:
Attempting to integrate autolog button feature

kaiyork committed:
Create Routes not worth pursuing md

kaiyork committed:
Create Abandoned routes.md

kaiyork committed:
Update Resources used by kai

kaiyork committed:
Update Resources used by kai

CamPBrown committed:
fixed the tracking button so it works

kaiyork committed:
Update Abandoned routes.md

old code for tracking location, replace by new one, kept here in case i need to revisit

kaiyork committed:
Update Resources used by kai

updated with new way of grabbing current location

kaiyork committed:
Update Abandoned routes.md

kaiyork committed:
Update Resources used by kai

CamPBrown committed:
Fixed an issue with the movement prompts not appearing when clicking the button

CamPBrown committed:
Fixed a tracking issue

CamPBrown committed:
Updated tracking prompts so they won't appear as soon as tracking is turned off by button

CamPBrown committed:
Added a login screen and login functionality

CamPBrown committed:
Login and sign out screens designed and implemented

CamPBrown committed:
Created Login and Sign Out screens

CamPBrown committed:
Merge pull request #41 from camtheboom/master

Master

CamPBrown committed:
Fixed issue with login page not displaying correctly on some phone screens

CamPBrown committed:
Merge branch 'main' of https://github.com/camtheboom/CITS3200-Team-4

camtheboom committed:
Format changes

camtheboom committed:
Attempt to connect to manualLog

camtheboom committed:
small design fix

camtheboom committed:
small fix

camtheboom committed:
Merge pull request #43 from camtheboom/42-manual-log-format

42 manual log format

CamFroth committed:
updated pages to use default.js well (div to div3)

kaiyork committed:
Update README.md

kaiyork committed:
Update App.js

kaiyork committed:
Update App.js

kaiyork committed:
Update Profile.js

added map feature to main app

CamFroth committed:
Changes to login page

CamFroth committed:
Updated splash screen and text

CamFroth committed:
Merge pull request #44 from camtheboom/camfroth-updateddesign

Updated Design for Login, Logout and Splash screen

kaiyork committed:
Merge pull request #45 from camtheboom/kai-mk2

Kai mk2

CamPBrown committed:
updated manual log so it is saved to database and retrieved from database

CamPBrown committed:
A more robust and better way to keep checking user position

CamPBrown committed:
Changed display of pages and removed un-used pages

CamFroth committed:
Added welcome modal

CamFroth committed:
Changing style of modal

CamPBrown committed:
manuallog is now stored as list of object of responses

CamPBrown committed:
fixed an issue with the placeholder text causing a crash on mobile

CamFroth committed:
Latest iteration of welcome modal

camtheboom committed:
Travel changes

camtheboom committed:
minor changes to app.js

camtheboom committed:
changes are for travellog screen styles

CamFroth committed:
Successful front-end implementation of
welcome modal
other design tweaks (button radius,
font weight,
general colour

CamFroth committed:
Changed grey buttons to dark blue

camtheboom committed:
small fix for travellog

CamFroth committed:
Added running man to home screen

CamFroth committed:
Added fun images to the functioning screens
Added credits to the login screen
Reworked dimensions of login screen to push buttons up

CamFroth committed:
Slightly adjusted button size to prevent clipping

CamFroth committed:
Merge branch 'main' into camfroth-welcomemodal

CamFroth committed:
Merge pull request #46 from camtheboom/camfroth-welcomemodal

Camfroth welcomemodal merged

CamPBrown committed:
working coordinate tracking, coordinates are now saved and used to determine if user has moved or stopped

CamPBrown committed:
Merge branch 'manual_log' of https://github.com/camtheboom/CITS3200-Team-4 into manual_log

CamPBrown committed:
Merge branch 'main' into manual_log

CamPBrown committed:
Merge pull request #47 from camtheboom/manual_log

Merged updated manual log and coordinate saving with main

CamPBrown committed:
fixed a minor issue

CamPBrown committed:
added backend to the personal data submission modal

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

kaiyork committed:
Update README.md

CamPBrown committed:
Lowered interval for logging/checking movement. Added an additional text input for current location. Added some more functionality to clear text input on modal pop-ups only when the user has exited them.

CamPBrown committed:
Merge branch 'main' of https://github.com/camtheboom/CITS3200-Team-4

CamFroth committed:
Added comments to styling features

CamFroth committed:
Added messages to autolog buttons
Fixed bug where only tapping on text started the autolog

CamFroth committed:
Added borders to input fields in movement modals

CamFroth committed:
Adjusted styling of manual log history

CamFroth committed:
added file to build app from
